internal/config/main.go

```diff
@@ -73,6 +73,20 @@ func EntraInviteRedirectURL() string {
 	return k.String("entra.invite_redirect_url")
 }
 
+func EntraTenantPrimaryDomain() string {
+	return k.String("entra.primary_domain")
+}
+
+func EntraInvitedUserGroup() string {
+	return k.String("entra.invited_user_group_id")
+}
+
+func EntraDomainForTesting() (string, error) {
+	if err := k.Set("entra.primary_domain", "testTenant.com"); err != nil {
+		return "", err
+	}
+	return k.String("entra.primary_domain"), nil
+}
```

Comment:
I think this would be better as 
```go

func SetForTesting(key string, value string) error {
	return k.Set(key, value)
}
```
then called from the test